### DISCUSSION TITLE
v1.0.0-beta.3

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#476] [General]
🦋 Follow IPTVnator on BlueSky: https://bsky.app/profile/iptvnator.bsky.social
💬 Discuss the app in Telegram: https://t.me/iptvnator

----

## What's Changed
* Fix for Dependency Injection Issue in DataFactory by @abdullahceylan in https://github.com/4gray/iptvnator/pull/474
* v1.0.0-beta.3 by @4gray in https://github.com/4gray/iptvnator/pull/471

**Fullscreen Capability**: 
   - Added the `core:window:allow-set-fullscreen` capability to the default configuration, enabling fullscreen mode for a more immersive user experience.

**Search Input Improvements**: 
   - Disabled spellcheck and autocomplete in the search input field within the channel list component, providing a cleaner and more focused search experience.

**Error Handling Enhancements**: 
   - Improved error handling and filtering logic in the `getPortalFavorites` and `getPortalLiveStreamFavorites` methods, including null checks to prevent runtime errors.

**EPG Data Fetching Implementation**: 
   - Added support for multiple EPG sources with a new `EpgStore` structure. Implemented the `fetch_and_parse_epg` function to handle data fetching from various sources, improving error handling and logging.

**Responsive Layout Adjustments**: 
   - Adjusted video player and layout heights for better responsiveness, ensuring a more user-friendly interface.

**Drag&Drop fix and refactor of Favorites List**: 
 - Simplified the channel list component's template structure, enhancing code readability and maintainability.

## New Contributors
* @abdullahceylan made their first contribution in https://github.com/4gray/iptvnator/pull/474

**Full Changelog**: https://github.com/4gray/iptvnator/compare/app-v1.0.0-2...app-v1.0.0-3

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/app-v1.0.0-3'>v1.0.0-beta.3</a>.</em>

**Comment by 13750985588** [comment]
iptvnator
